### PR TITLE
Add executive param for local test

### PR DESCRIPTION
**Pull request by AsceticBear**
I encountered https://github.com/paritytech/subport/issues/199 this issue again in local test cases. Update the script to avoid others down the same rabbit hole.

### CODE REVIEW

**Comment by AurevoirXavier** [comment]
A flag would be better.
`--execution` native will fail the runtime upgrade testing.

**Comment by AsceticBear** [comment]
> A flag would be better. `--execution native` will fail the runtime upgrade testing.

Make sense. Add a command flag, the default execution is `native`~~

**Comment by AsceticBear** [comment]
@AurevoirXavier Do you know what happened to the ci pipe?

**Comment by AurevoirXavier** [comment]
Ignored. This script doesn't affect the runtime/node/test behavior.

File: tests/local-testnet/deploy.sh [hunk 1]
```diff
@@ -49,6 +54,7 @@ do
     --unsafe-ws-external \
     --rpc-methods unsafe \
     --rpc-cors all \
+    --execution $EXECUTION \
```
Comment (AurevoirXavier) [inline tests/local-testnet/deploy.sh:57]: What if `$EXECUTION` is empty

Comment (AsceticBear) [reply tests/local-testnet/deploy.sh:57]: `$EXECUTION` comes from command, If empty, line 16~18, `$EXECUTION=native`

Comment (AurevoirXavier) [reply tests/local-testnet/deploy.sh:57]: Empty should be default behavior.

Comment (AsceticBear) [reply tests/local-testnet/deploy.sh:57]: @AurevoirXavier Please review again

**Pull request merged**